src/Microsoft.Azure.WebJobs.Host/Bindings/SysBindingData.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Microsoft.Azure.WebJobs.Host.Bindings.Path;
+
+namespace Microsoft.Azure.WebJobs.Host.Bindings
+{
+    /// <summary>
+    /// Built in object for all binding expressions. 
+    /// </summary>
+    /// <remarks>
+    /// It's expected this class is created and added to the binding data. 
+    /// </remarks>
+    internal class SysBindingData
+    {
+        // The name for this binding in the binding expressions. 
+        public const string Name = "sys";
+
+        // A name that can't be overwritten by the user that we cna place in the binding data dictionary
+        // and use to unambiguously retrieve this later. 
+        private const string InternalName = "$sys";
```

Comment:
Also, I recommend we use MS_, like we do for all our system http request properties (e.g. see https://github.com/Azure/azure-webjobs-sdk-script/blob/0c2703b116456b4764d6082d74541e18a60ce04b/src/WebJobs.Script/ScriptConstants.cs#L11)